<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_nomina_cargos</class>
 <widget class="QWidget" name="Form_nomina_cargos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cargos Cooperativa</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Courier 10 Pitch&quot;;
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(170, 255, 127);
color: rgb(0, 0, 0);
background-color: rgb(117, 0, 88);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(226, 226, 226);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Cargar cargos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Apellido :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_apellido">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="boton_cargo_buscar">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(140, 189, 58);</string>
           </property>
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>N° Documento:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_cargo_nro_doc_buscar">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>N° asociado:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_nro_asoc">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Apellido y nombre:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_apellido_nombre">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Documento:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbx_tipo_doc">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <item>
            <property name="text">
             <string>DNI</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CI</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_nro_doc">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Fecha nacimiento:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dte_fec_nac">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Cargo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbx_cargo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <item>
            <property name="text">
             <string>Presidente</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vicepresidente</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Secretario/a</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tesorero/a</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Síndico Titular</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Síndico Suplente</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vocal 1°</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Desde:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dte_desde">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Hasta:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dte_hasta">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Observaciones:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_observacion">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="boton_cargo_agregar">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(140, 189, 58);</string>
         </property>
         <property name="text">
          <string>Agregar</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QTableWidget" name="tw_nomina_cargos">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <column>
          <property name="text">
           <string>Apellido y nombre</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>DNI</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>N° asociado</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cargo</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Desde</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hasta</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Observaciones</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="background">
           <color>
            <red>177</red>
            <green>215</green>
            <blue>154</blue>
           </color>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="boton_limpiar">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(140, 189, 58);</string>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_observacion</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_nro_doc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_apellido_nombre</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_nro_asoc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_cargo_nro_doc_buscar</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_apellido</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
